content/20.pharmaceuticals.md

```diff
@@ -339,8 +339,13 @@ However, the relative success of remdesivir compared to favipiravir underscores
 ### Computational Approaches to Designing or Identifying Therapeutics
 
 As illustrated by the difference in the current status of remdesivir and favipiravir as treatments for COVID-19, identifying specific compounds that will produce a desired interaction with a virus is challenging.
-<!-- To Do: 1 paragraph background on how computational prediction works and why it's feasible now than in 2002 or 2011-->
-<!-- To Do: Is this primarily small molecule or also biologics? Need to be explicit about this-->
+Additionally, the drug development process is slow and costly, meaning that the development of compounds specifically targeted to an emerging viral threat are not a practical solution in the short term.
+As a result, in recent years, efforts to screen existing drug compounds for alternative indications have become popular [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1016/j.drudis.2019.06.014; @doi:10.1111/bph.13895].
+While some efforts to screen for potential repurposing opportunities are experimental, others use high-throughput computational approaches [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1016/j.tim.2018.04.004].
+For infectious viral diseases, several strategies can be employed: identifying drugs that interact with a target that is shared between a novel and known viral pathogen, identifying drugs that interact with a target that is shared between a novel viral pathogen and another illness (i.e., a cancer drug with antiviral potential), and identifying a drug that might interact with additional molecular targets beyond those it was developed for [@doi:10.1016/j.tim.2018.04.004].
```

Comment:
Just to clarify these different strategies. Is the first approach to identify a target that is shared between say SARS-CoV-2 and SARS-CoV (or some other virus)? Whereas the second approach is to identify a target that is shared between COVID-19 vs cancer or some other disease? So its a difference between looking at the direct target of a pathogen vs the downstream changes of an illness?

Reply:
That is my understanding! The description of these categories is below figure one in [this paper](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(18)30087-8).
The example they give is cellular Abelson (ABL) kinase which is a target of cancer drugs but can also be a target for interfering with coronaviruses.